.github/workflows/dast-release.yml

```
@@ -236,49 +227,47 @@ jobs:
-            image_digest: ${{ inputs.image_digest-frontend }}
+            image_digest: ${{ inputs.image-digest-frontend }}
     steps:
       - name: Download DAST artifacts
         uses: actions/download-artifact@3e5f45b2cfb9172054b4087a40e8e0b5a5461e7c # v4.1.8.0.1
         with:
           name: zap-results
           path: zap-downloads
-
       - uses: docker/login-action@4907a6ddec9925e35a0a9e82d7399ccc52663121 #v4.1.0
         with:
           username: ${{ secrets.dockerhub_username }}
```

actionlint (1.7.12)

[error] property "dockerhub_username" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; admin_pass: string; data_encryption_key: string; db_pass: string; dockerhub_token: string; ghcr_token: string; github_token: string; jwt_secret: string}

(expression)